### ISSUE TITLE
Modify S6330: Default Queue encryption is now SSE-SQS

### ISSUE THREAD

**Issue by sebastien-andrivet-sonarsource** [SonarSource/SonarJS#4035]
# What:
This rule detects the usage of no encryption when creating Queues. Currently, it raises false positives when no encryption is specified or when SSE-SQS encryption is used.

# Detection Logic:
This issue should not be raised when SSE-SQS or SS-KMS encryption is specified. When no encryption is explicitly specified, SSE-SQS encryption is used by default.

## Fully qualified name:
* `aws-cdk-lib.aws-sqs.Queue`
* `aws-cdk-lib.aws-sqs.CfnQueue`

## Raise an issue if:
* Raise an issue if `sqsManagedSseEnabled` is set to `false`

## Do not raise an issue if:
* Do not raise an issue if `sqsManagedSseEnabled` is set to `true`
* Do not raise an issue if `kmsMasterKeyId ` is set
* Do not raise an issue if `sqsManagedSseEnabled` and `kmsMasterKeyId ` are missing

# Examples:

## Noncompliant

```typescript
const cfnQueue = new CfnQueue(this, 'CfnQueueNoEncryption', {
      sqsManagedSseEnabled: false // Sensitive
    });
```

## Compliant

```typescript
const cfnQueue = new CfnQueue(this, 'CfnQueueEncrypted', {
      sqsManagedSseEnabled: true // Encrypted with SSE-SQS
    });
```

# SPEC
The relevant RSPEC is: https://sonarsource.atlassian.net/browse/SONARIAC-1030

**Comment by Wohops** [comment]
Migrated to Jira. New ticket number: JS-82
